Sources/NostrSDK/Events/Calendars/DateBasedCalendarEvent.swift

```diff
@@ -28,7 +28,7 @@ public final class DateBasedCalendarEvent: NostrEvent, CalendarEventInterpreting
     /// Start date is represented by ``TimeOmittedDate``.
     /// `nil` is returned if the backing `start` tag is malformed.
     public var startDate: TimeOmittedDate? {
-        guard let startString = tags.first(where: { $0.name.rawValue == "start" })?.value else {
+        guard let startString = tags.first(where: { $0.name == "start" })?.value else {
```

Comment:
It might be outside of the scope of this PR, but I wonder if it would be useful to have bespoke tag name enums scoped down to each event kind level. That way we're not dealing with raw strings everywhere, and it can be shared between `EventCreating` and the NostrEvent kind class.